get_contact.py

```diff
@@ -0,0 +1,46 @@
+import base64
+import json
+import requests
+import sys
+
+def get_readme(owner, repo):
+    repo_name = owner + '/' + repo
+    url = 'https://api.github.com/repos/' + repo_name + '/readme'
+
+    r = requests.get(url)
```

Comment:
Every time a `request` is done, please check for server/connection errors with `r.raise_for_status()`.